plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -113,12 +128,82 @@ class WindowSizeChannel {
   void setWindowFrame(Rect frame) async {
     assert(!frame.isEmpty, 'Cannot set window frame to an empty rect.');
     assert(frame.isFinite, 'Cannot set window frame to a non-finite rect.');
-    try {
-      await _platformChannel.invokeMethod(_setWindowFrameMethod,
-          [frame.left, frame.top, frame.width, frame.height]);
-    } on PlatformException catch (e) {
-      print('Platform exception setting window frame: ${e.message}');
+    await _platformChannel.invokeMethod(_setWindowFrameMethod,
+        [frame.left, frame.top, frame.width, frame.height]);
+  }
+
+  // Window minimum size unconstrained is passed over the channel as -1.
+  double _setWindowMinSizeUnconstrainedCheck(double size) {
+    if (size > 0) {
+      return size;
     }
+    return -1;
+  }
+
+  /// Sets the minimum size of the window containing this Flutter instance.
+  void setWindowMinSize(Size size) async {
+    await _platformChannel.invokeMethod(_setWindowMinimumSizeMethod, [
+      _setWindowMinSizeUnconstrainedCheck(size.width),
+      _setWindowMinSizeUnconstrainedCheck(size.height),
+    ]);
+  }
+
+  // Window maximum size unconstrained is passed over the channel as -1.
+  double _setWindowMaxSizeUnconstrainedCheck(double size) {
```

Comment:
This method isn't setting anything, and I'm not sure what `check` is supposed to communicate exactly, so this name doesn't seem to me to clearly communicate what the method does. I would call this something like `_channelRepresentationForMaxDimension` (I wouldn't use Size since it doesn't actually take a size)